andela/ah-frontend-targaryen
# #161776762 Edit articles

### What does this PR do?

Enables users to edit articles which they authored

#### Description of Task to be completed?

Currently, the users are unable to update articles. 

This feature enables users to edit articles. Users can only edit the articles they authored.

#### How should this be manually tested?

Pull the changes on a new branch and run `npm install`
run `npm start` to start the server
Log into the app using a registered email and password
Click on profile and select `New Article` to add a new article
Select the created article from the list of articles by clicking on `Read more` link
On loading the article, click on edit button and change any of the fields
Click on save button and see the updated article on the profile


#### What are the relevant pivotal tracker stories?

- [#161776762](https://www.pivotaltracker.com/story/show/161776762)

#### Screenshots (if appropriate)


## Checklist: I completed these to help reviewers
